java/src/test/java/com/logicalclocks/hsfs/TestHopsworksExternalClient.java

```diff
@@ -53,6 +53,21 @@
         .willReturn(success().body(HttpBodyConverter.json(credentials)))
     ));
 
+  @Test
+  public void testReadAPIKey() throws IOException, FeatureStoreException {
```

Comment:
The options I see are:
1. comment it out and leave the code there, so it can easily be tested again
2. delete the code.

What do you think?

Reply:
We can leave it commented out, but I still believe the `hc.getFeatureStore()` will never return null but instead throw a exception if it can't get the feature store reference and hence the test will never get to the assert and always pass as the exception gets caught by the catch statement.